service/device/registry/api/src/main/java/org/eclipse/kapua/service/device/registry/KapuaDeviceRegistrySettingKeys.java

```diff
@@ -13,41 +13,61 @@
 package org.eclipse.kapua.service.device.registry;
 
 import org.eclipse.kapua.commons.setting.SettingKey;
+import org.eclipse.kapua.message.device.lifecycle.KapuaBirthMessage;
 
 /**
- * Available settings key for user service
- *
- * @since 1.0
+ * {@link SettingKey}s for KapuaDeviceRegistrySettings.
  *
+ * @since 1.0.0
  */
 public enum KapuaDeviceRegistrySettingKeys implements SettingKey {
 
     /**
      * The key value in the configuration resources.
+     *
+     * @since 1.0.0
      */
-    DEVICE_REGISTRY_KEY("device_registry.key"),
-    DEVICE_EVENT_ADDRESS("device.eventAddress");
+    DEVICE_REGISTRY_KEY("device.registry.key"),
 
-    private String key;
+    /**
+     * @since 1.0.0
+     */
+    DEVICE_EVENT_ADDRESS("device.eventAddress"),
 
     /**
-     * Set up the {@code enum} with the key value provided
+     * Gets the hard limit for {@link KapuaBirthMessage} generic fields length.
      *
-     * @param key
-     *            The value mapped by this {@link Enum} value
+     * @since 2.0.0
+     */
+    DEVICE_REGISTRY_LIFECYCLE_BIRTH_FIELDS_CLOB_LENGTH_MAX("device.registry.lifecycle.birth.fields.clob.length.max"),
```

Comment:
For this enum item I would use a name that it is a bit more consistent with the next one and maybe more specific. Something like:

- DEVICE_LIFECYCLE_BIRTH_VAR_FIELDS_LENGTH_MAX (or similar)
or

- DEVICE_LIFECYCLE_BIRTH_CONNECTION_IP_LENGTH_MAX
- DEVICE_LIFECYCLE_BIRTH_CONNECTION_INTERFACE_LENGTH_MAX

Reply:
Changed to be `DEVICE_LIFECYCLE_BIRTH_VAR_FIELDS_LENGTH_MAX`

Reply:
Changed to be `DEVICE_LIFECYCLE_BIRTH_VAR_FIELDS_LENGTH_MAX`